### PR TITLE
Add Alnair entry to releases.json

### PR DESCRIPTION
**Pull request by ajsantander**
Also makes a slight modification to the prepare-deploy script, allowing it to pick a version depending on the evm/ovm nature of the release.

### CODE REVIEW

File: publish/src/commands/prepare-deploy.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const prepareDeploy = async ({ network = DEFAULTS.network, useOvm }) => {
 	const config = JSON.parse(fs.readFileSync(configFile));
 
 	// Pick the latest release from the list
-	const release = releases.slice(-1)[0];
+	const release = releases.reverse().find(release => (useOvm ? release.ovm : !release.ovm));
```
Comment (justinjmoses) [inline publish/src/commands/prepare-deploy.js:30]: Yeah slick. I like it.

File: publish/releases.json [hunk 2]
```diff
@@ -184,5 +184,24 @@
...
+		"sips": [116]
+	},
+	{
+		"name": "Alnair (Optimism)",
+		"version": {
+			"major": 2,
+			"minor": 43
+		},
+		"ovm": true,
+		"sources": ["SynthetixBridgeToBase", "Synthetix", "DebtCache"],
```
Comment (justinjmoses) [inline publish/releases.json:204]: Hmmm - what about `SystemStatus`? I think we should deploy the latest version from Bellatrix to L2 which was https://sips.synthetix.io/sips/sip-109

**Review by justinjmoses — CHANGES_REQUESTED** [review]
Just an additional contract to L2.

File: publish/releases.json (hunk 2 above)
Comment (ajsantander) [reply publish/releases.json:204]: Ah yeah, good call.

**Review by justinjmoses — APPROVED** [review]

**Review by i-stam — APPROVED** [review]
lgtm, just a reminder, we should add the SynthexBridgeEscrow contract when sip-116 is merged.

**Comment by justinjmoses** [comment]
Damn - gonna have to go back to *Alnair*. *Regora* is going to be used for the sCOIN and sMSFT releases.

**Review by justinjmoses — CHANGES_REQUESTED** [review]
Gonna have to use *Alnair* now

File: publish/releases.json [hunk 3]
```diff
@@ -184,5 +184,29 @@
...
+		"name": "Regora (Optimism)",
+		"version": {
+			"major": 2,
+			"minor": 43
+		},
+		"ovm": true,
+		"sources": [
+			"SynthetixBridgeToBase",
+			"Synthetix",
+			"DebtCache",
```
Comment (justinjmoses) [inline publish/releases.json:207]: You need to also add `Exchanger` - it's for SIP-118 - and add that below as well

Comment (ajsantander) [reply publish/releases.json:207]: Oh wait, now we see what you mean. Yeah, adding it.

File: publish/releases.json [hunk 4]
```diff
@@ -184,5 +184,30 @@
 		},
 		"sources": ["Exchanger", "VirtualSynthMastercopy"],
 		"sips": [107, 114, 126, 127]
+	},
+	{
+		"name": "Alnair",
```
Comment (justinjmoses) [inline publish/releases.json:189]: Please make sure #1227 is merged first and then add this after.

Comment (ajsantander) [reply publish/releases.json]: Done, updated this branch to develop and changed Alnair's minor to 44

**Review by justinjmoses — APPROVED** [review]

**Pull request merged**